<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1050</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1050</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>学生管理系统</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" rowspan="3">
     <widget class="myTable" name="table">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>550</width>
        <height>542</height>
       </size>
      </property>
      <property name="toolTip">
       <string>显示学生数据的表单</string>
      </property>
      <property name="statusTip">
       <string>显示学生数据的表单</string>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn_confirm">
      <property name="toolTip">
       <string>添加数据</string>
      </property>
      <property name="statusTip">
       <string>添加数据，可添加到所选行的后面</string>
      </property>
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QCheckBox" name="editBox">
      <property name="toolTip">
       <string>设置可否编辑状态</string>
      </property>
      <property name="statusTip">
       <string>设置可否编辑状态</string>
      </property>
      <property name="text">
       <string>是否可编辑</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn_delete">
      <property name="toolTip">
       <string>删除数据</string>
      </property>
      <property name="statusTip">
       <string>删除数据，可多选数据删除</string>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>btn_confirm</zorder>
   <zorder>btn_delete</zorder>
   <zorder>table</zorder>
   <zorder>editBox</zorder>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1050</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="newFile"/>
    <addaction name="openFile"/>
    <addaction name="saveFile"/>
    <addaction name="saveAsFile"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>排序</string>
    </property>
    <addaction name="sortbyNumber"/>
    <addaction name="sortbyAge"/>
    <addaction name="sortbySex"/>
    <addaction name="sortbyClass"/>
    <addaction name="sortbyName"/>
    <addaction name="separator"/>
    <addaction name="reverse"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>个性化</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>设置颜色</string>
     </property>
     <addaction name="setTextColor"/>
     <addaction name="setColor"/>
    </widget>
    <addaction name="setFont"/>
    <addaction name="menu_3"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>查找</string>
    </property>
    <addaction name="findByName"/>
    <addaction name="findByNumber"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menu"/>
   <addaction name="menu_4"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="openFile">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="iconText">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>打开一个现有的文件</string>
   </property>
   <property name="statusTip">
    <string>打开一个现有的文件，也可以通过拖动打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>true</bool>
   </property>
  </action>
  <action name="saveAsFile">
   <property name="text">
    <string>另存为</string>
   </property>
   <property name="toolTip">
    <string>文件另存为</string>
   </property>
   <property name="statusTip">
    <string>文件另存为</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="newFile">
   <property name="text">
    <string>新建</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="sortbyNumber">
   <property name="text">
    <string>按学号排序</string>
   </property>
  </action>
  <action name="sortbyAge">
   <property name="text">
    <string>按年龄排序</string>
   </property>
  </action>
  <action name="sortbySex">
   <property name="text">
    <string>按性别排序</string>
   </property>
  </action>
  <action name="sortbyName">
   <property name="text">
    <string>按照姓名排序</string>
   </property>
  </action>
  <action name="reverse">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>反向</string>
   </property>
   <property name="toolTip">
    <string>是否反向排序当前数据</string>
   </property>
  </action>
  <action name="sortbyClass">
   <property name="text">
    <string>按班级排序</string>
   </property>
  </action>
  <action name="setFont">
   <property name="text">
    <string>设置字体</string>
   </property>
  </action>
  <action name="setColor">
   <property name="text">
    <string>设置背景颜色</string>
   </property>
  </action>
  <action name="setTextColor">
   <property name="text">
    <string>设置字体颜色</string>
   </property>
  </action>
  <action name="findByName">
   <property name="text">
    <string>按照姓名</string>
   </property>
  </action>
  <action name="findByNumber">
   <property name="text">
    <string>按照学号</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>myTable</class>
   <extends>QTableView</extends>
   <header>mytable.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
